pyscriptjs/tests/integration/conftest.py

```diff
@@ -130,6 +130,13 @@ class HTTPServer(SuperHTTPServer):
     Keyboard Interrupt passes.
     """
 
+    def __init__(self, host_and_port, handler_class):
+        self.handler_class = handler_class
```

Comment:
this is superfluous. `HTTPServer` is a subclass of `socketserver.SocketServer`, and the class is already stored as `RequestHandlerClass`:
https://github.com/antocuni/cpython/blob/deaf509e8fc6e0363bd6f26d52ad42f976ec42f2/Lib/socketserver.py#L201-L206

Reply:
thanks, done